config/initializers/content_security_policy.rb

```diff
@@ -7,8 +7,8 @@
 Rails.application.config.content_security_policy do |policy|
 #   policy.default_src :self, :https, 'youtube.com', 'www.youtube.com'
 #   policy.font_src    :self, :https, :data, 'fonts.gstatic.com'
-#   policy.img_src     :self, :https, :data, 'view.vzaar.com', 'blog.teachcomputing.org', 'futurelearn-assets.raspberrypi.org'
-#   policy.media_src   :self, :https, :data, 'view.vzaar.com', 'download.vzaar.com', 'futurelearn-assets.raspberrypi.org'
+#   policy.img_src     :self, :https, :data, 'view.vzaar.com', 'blog.teachcomputing.org'
+#   policy.media_src   :self, :https, :data, 'view.vzaar.com', 'download.vzaar.com'
```

Comment:
What is the value in these commented out lines remaining in the project? There isn't a comment explaining why they're here or how to use them - perhaps that is worth adding if they're valuable?

Reply:
It was temporarily disabled 2 years ago. https://github.com/NCCE/teachcomputing.org/pull/1160
I'll add a comment.